src/TipBox.vue

```diff
@@ -50,12 +65,15 @@
         type: String,
         default: ''
       },
-      heading: {
+      header: {
```

Comment:
Potential breaking change (some people might have already started using the new `heading` feature, which means their site will break immediately). My suggestion is we support both first while deprecating `heading`.